Sources/SwiftFormatPrettyPrint/PrettyPrint.swift

```diff
@@ -275,10 +278,18 @@ public class PrettyPrinter {
         // lines within it (unless they are themselves continuations within that particular
         // scope), so we need the continuation indentation to persist across all the lines in that
         // scope.
-        activeOpenBreaks.append(
-          ActiveOpenBreak(
-            lineNumber: currentLineNumber,
-            contributesContinuationIndent: currentLineIsContinuation))
+        var activeOpenBreak = ActiveOpenBreak(
+          kind: openKind,
+          lineNumber: currentLineNumber,
+          contributesContinuationIndent: currentLineIsContinuation)
+        activeOpenBreak.contributesBlockIndent = openKind == .block
```

Comment:
Let's go ahead and make this an initializer argument too, since it's always set immediately after creating the instance.